src/data.js

```diff
@@ -1,40 +1,46 @@
 export const animals = [
   {
+    id: 1,
```

Comment:
nice work on identifying that the key could be correlated to an id, you can have also used the name in this case.